syntax_and_semantics/methods_and_instance_variables.md

```diff
@@ -44,6 +44,26 @@ peter.age #=> 0
 
 Method names begin with a lowercase letter and, as a convention, only use lowercase letters, underscores and numbers.
 
+## Getters and setters
+
+The Crystal [Standard Library](https://crystal-lang.org/api) provides macros which ease the definition of getter and setter methods:
+
+```crystal
+class Person
+  getter :age
```

Comment:
Please don't use symbol literals, the convention is to use bare names: `property age`.

Reply:
Sorry, I didn't see anything in the style guide so I picked one. Other than [the gitbook style guide](https://crystal-lang.org/docs/conventions/coding_style.html) is there another place where I should be checking?